### PR TITLE
There shall be only one logger

### PR DESCRIPTION
**Pull request by jrouzierinverse**
## Description

Remove the use of two seperate logging subsystem
## Impacts

Logging in pfconfig namespace
## NEWS file entries

New Features
++++++++++++

Enhancements
++++++++++++

Bug Fixes
+++++++++
## Delete branch after merge

NO

### CODE REVIEW

**Comment by jrouzierinverse** [comment]
This also slightly improves the performance of pfconfig.

File: lib/pfconfig/manager.pm [hunk 1]
```diff
@@ -197,26 +200,18 @@ That sends the signal that the raw memory is expired
...
     $filename = untaint_chain($filename);
 
-    if ( !-e $filename ) {
-        my $fh;
-        unless ( open( $fh, ">$filename" ) ) {
-            $logger->error("Can't create $filename\nPlease run 'pfcmd fixpermissions'");
-            return 0;
-        }
+    if (sysopen(my $fh,$filename,O_RDWR | O_CREAT)) {
+        POSIX::2008::futimens(fileno $fh);
```
Comment (julsemaan) [inline lib/pfconfig/manager.pm]: I had an issue with this, this is why it was changed to the ugly touch command.

There were a few microseconds of difference between this and the stat in Time::HiRes which made unit tests fail

**Comment by julsemaan** [comment]
Other than my comment on the replacement of the file touching, this is all good

File: lib/pfconfig/manager.pm (hunk 1 above)
Comment (rouzier) [inline lib/pfconfig/manager.pm]: Ok I normalized POSIX::2008::futimens to use the same resolution of Time::HiRes.
My changes have been pushed please re-review

Comment (jrouzierinverse) [inline lib/pfconfig/manager.pm]: All the test are passing now

**Pull request merged**